libs/dotcms-js/src/lib/core/site.service.ts

```diff
@@ -156,7 +156,9 @@ export class SiteService {
                 method: 'PUT',
                 url: `${this.urls.switchSiteUrl}/${site.identifier}`
             })
-            .subscribe();
+            .subscribe(() => {
```

Comment:
This looks like something that needs a take 1

Reply:
@fmontes done!